src/appleseed/renderer/modeling/shadergroup/shadergroup.h

```diff
@@ -151,17 +154,19 @@ class APPLESEED_DLLSYMBOL ShaderGroup
         HasHoldout      = 1 << 4,
```

Comment:
Since holdout is a special case of Matte, lets remove HasHoldout and treat holdouts as matte.